deutsches-archaologisches-institut.csl

```diff
@@ -412,12 +412,13 @@
       <key variable="issued"/>
       <key macro="contributors-short-citation"/>
     </sort>
-    <layout prefix="" suffix="" delimiter="; ">
+    <layout delimiter="; ">
```

Comment:
I'm actually wondering -- why aren't there parentheses here? That seems odd -- surely the style doesn't just put Author 1999 in the text?